common/api-gateway.conf.in

```diff
@@ -22,7 +22,7 @@
 # */
 #user                 nginx-api-gateway;
 user                 root;
-worker_processes     auto;
+worker_processes     ${worker_processes};
```

Comment:
what's the reason for not leaving this to `auto` ?

Reply:
When building a profiling image, this value is set to 1 to support profiling.  The variable lets that be set at build time.